sdk/communication/Azure.Communication.Chat/src/autorest.md

```diff
@@ -11,7 +11,7 @@ If any of the new objects needs to be overwritten, add the required changes to t
 
 ``` yaml
 input-file:
-    -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/5c659a0b9a2826f1133cef96748f5c7b956557bf/specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json
+    -  communicationserviceschat.json
```

Comment:
Lets point to the preview4 url and remove the file from here (if the file is not check with @amr if that version is in the azure-rest-api-specs repo, lets ping Amr to add it)

Reply:
Currently there are only preview 2 and preview 3 json files in the repo (https://github.com/Azure/azure-rest-api-specs/tree/master/specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview), I will ask Amr about adding preview 4.